internal/utils_common/files_test.go

```diff
@@ -19,6 +20,130 @@ type fileDetail struct {
 	Name string
 }
 
+// FileRemover defines the interface for removing files.
+type FileRemover interface {
+	RemoveAll(path string) error
+}
+
+// deletePaths removes paths using the provided FileRemover.
+func deletePaths2(paths []string, remover FileRemover) (int, error) {
+	deletedCount := 0
+	for _, path := range paths {
+		if err := remover.RemoveAll(path); err != nil {
+			return deletedCount, fmt.Errorf("failed to remove %s: %v", path, err)
+		}
+		deletedCount++
+	}
+	return deletedCount, nil
+}
```

Comment:
not sure what deletePaths**2** is?

Reply:
oh sorry... delete paths 2 and some other code above it were actually not meant to be a part of the push. They are now removed.